### PR TITLE
Update Main Config File

### PR DESCRIPTION
**Pull request by jokoho48**
after Bux Commit in Slack

### CODE REVIEW

File: addons/main/config.cpp [hunk 1]
```diff
@@ -541,26 +541,28 @@ class CfgPatches {
...
+        dir = "@ACE3";
+        name = "Advanced Combat Environment 3";
         picture = "A3\Ui_f\data\Logos\arma3_expansion_alpha_ca";
         hidePicture = "true";
         hideName = "true";
         actionName = "Website";
-        action = "http://ace.dev-heaven.net";
-        description = "Bugtracker: ";
+        action = "http://ace3mod.com/";
+        description = "Bugtracker: https://github.com/acemod/ACE3/issues";
```
Comment (jonpas) [inline addons/main/config.cpp]: `"Issue Tracker: <link>"` rather? To be consistent with everywhere else.

File: addons/main/config.cpp [hunk 2]
```diff
@@ -541,26 +541,28 @@ class CfgPatches {
...
+        versionAct = "";
         VERSION_CONFIG;
     };
 };
 
 class CfgMods {
     class PREFIX {
-        dir = "@ACE";
-        name = "Core - Advanced Combat Environment";
+        dir = "@ACE3";
```
Comment (jonpas) [inline addons/main/config.cpp]: Directory is actually `@ace`.

File: addons/main/config.cpp [hunk 3]
```diff
@@ -541,26 +541,28 @@ class CfgPatches {
...
             "map_vr",
-            "extended_eventhandlers", "CBA_UI", "CBA_XEH", "CBA_XEH_A3"
+            "extended_eventhandlers", "CBA_UI", "CBA_XEH", "CBA_XEH_A3",
+            "ace_common"
         };
-        author[] = {"ACE Team"};
-        authorUrl = "";
-        versionDesc = "A.C.E.";
-        versionAct = QUOTE(['MAIN',_this] execVM '\z\ace\addons\main\about.sqf';);
+        author[] = {ELSTRING(common,ACETeam)};
```
Comment (jonpas) [inline addons/main/config.cpp]: Pretty sure you'll need `ECSTRING` (difference from L is that it adds a `$` in front which is needed in configs).

File: addons/main/config.cpp (hunk 2 above)
Comment (bux578) [inline addons/main/config.cpp]: I think `ACE` is fine here.

Comment (jokoho48) [inline addons/main/config.cpp]: the folder name is @ace @bux578 i think @jonpas  are right with that

File: addons/main/config.cpp (hunk 3 above)
Comment (jokoho48) [inline addons/main/config.cpp]: 😬

File: addons/main/config.cpp (hunk 2 above)
Comment (bux578) [inline addons/main/config.cpp]: yeah that's what i meant

Comment (SzwedzikPL) [inline addons/main/config.cpp]: what is purpose of `dir` ?

Comment (jokoho48) [inline addons/main/config.cpp]: directory short form is dir

**Pull request merged**